src/Eto/Forms/Controls/DocumentControl.cs

```diff
@@ -441,6 +575,66 @@ public new interface IHandler : Container.IHandler
 		/// </summary>
 		/// <value><c>true</c> to allow reordering; otherwise, <c>false</c>.</value>
 		bool AllowReordering { get; set; }
+
+		/// <summary>
+		/// Gets or sets a value indicating the tabs can be navigated by next and previous buttons.
+		/// </summary>
+		/// <value><c>true</c> to allow navigating; otherwise, <c>false</c>.</value>
+		bool AllowNavigating { get; set; }
```

Comment:
Even if `AllowNavigating` is set to `false` the control still allows to navigate, which makes the name of this property a bit misleading. How about renaming to something more specific, like `NavigationButtonsVisible`?

Reply:
Or maybe `HasNavigationStepper`, What do you think @ievgen-baida ?

Reply:
`HasNavigationButtons` is simpler.

Reply:
Renamed